README.md

```diff
@@ -1 +1,3 @@
-# CoNLL-U_cycles
\ No newline at end of file
+# CoNLL-U_cycles
+## Summary
+This program takes a CoNLL-U file and finds the cycles in this graph.
```

Comment:
What is this "graph"? (Although I know what you're talking about, it would not be clear to someone else who maybe doesn't know this terminology.) The best thing would be to give an example of what a cyclic graph is and what a non-cyclic graph is.

Reply:
Should I Give the Math Definition I Learned In Graph Theory?

Reply:
Actually in this context dependencies have directions so ill have to be more specific